Testing/Functional/Products/Tests/CallProductTests.ps1

```diff
@@ -0,0 +1,23 @@
+# James Garriss
+# Dec 2023
+# The purpose of this script is to run the functional tests for all the products.
+# NOTE: At the moment it only does this for one product, Sharepoint.  Others
+#       will be added.
+# To run this script, call it from the root of the repo, like so:
+# ./Testing/Functional/Products/Tests/CallProductTests.ps1
+
+# Setup directories
+$testDataFile = 'Testing/Functional/Products/Tests/TestData/sharepoint-commercial-data.pson'
```

Comment:
Interesting.  I would have expected this to be a parameter that is passed into this script (from user or workflow).

Reply:
If you read the TODO comment in the script, you will see that this is hard-coded for the first version.  The next US (731) will add tests for additional products.  As it says, the plan is to read all the PSON files in the directory and test them all.  We have here a to-MAY-to vs to-MAH-to thing.  Do you want the workflow to have to keep track of all possible tests and call them one by one?  Or do you want the script to find all PSON files on its own and run those tests?  I think I prefer the later, but neither is really wrong.  If you strong feel like it should do the former, please create a new conversation, and I will re-write it accordingly.